index.html

```diff
@@ -69,23 +69,140 @@ <h1 class="header">AnySoftKeyboard</h1>
 <a name="languages" class="anchor" href="#languages"><span class="octicon octicon-link"></span></a>Languages</h2>
 
 <p>The following languages with multiple dictionaries, auto-correction and keyboards are supported:</p>
-<ul>
-<li>English (QWERTY, DVORAK, AZERTY and Colemak)</li>
-<li>Arabic</li>
-<li>Belorussian</li>
-<li>Bulgarian</li>
-<li>Catalan</li>
-<li>Dutch</li>
-<li>German</li>
-<li>Hebrew</li>
-<li>Lao</li>
-<li>Portuguese</li>
-<li>Russian</li>
-<li>Spanish</li>
-<li>Swedish</li>
-<li>Swiss</li>
-<li>Ukrainian</li>
-</ul>
+<table>
+<tr><td>English (QWERTY, DVORAK, AZERTY and Colemak)</td>
+<td>built-in</td>
+<td>built-in</td>
+<td><a href="https://github.com/AnySoftKeyboard/AnySoftKeyboard"><img src="images/github.png" /></a></td></tr>
+<tr><td>Hacker's Keyboard (SSH)</td>
+<td><a href="https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard"><img src="images/play.png" /></a></td>
```

Comment:
Why is there a link to Hacker's keyboard here? It a full keyboard app, bot a pack (as far as I know)